docs/docs/reference/shards/built-ins/data-construction.md

```diff
@@ -33,7 +33,7 @@ Include `@i2`, `@i3`, `@i4`, `@i8`, `@i16`
     === "Expression Position"
     
         ```shards
-        Math.Add(Operand: @i2(1 1)) ;; compile time evaluation
+        Math.Add(Operand: @i2(1 1)) ;; compose time evaluation
```

Comment:
not sure what this means, there is no compose evaluation, it's just a constant i2?